Dockerfile

```diff
@@ -38,9 +38,9 @@ WORKDIR /app
 COPY LICENSE .en[v] config.py main.py messages.json ./
 RUN chmod +x main.py
 
-COPY cogs/ ./cogs/
-COPY db/ ./db/
-COPY utils/ ./utils/
 COPY exceptions/ ./exceptions/
+COPY utils/ ./utils/
+COPY db/ ./db/
+COPY cogs/ ./cogs/
```

Comment:
The order dictates how docker splits cached layers. I've tried to order them so that the layers that are most likely to be edited most frequently are added last so that the other layers do not need to be rebuilt and can rely on the caches, if for example only a new cog is added.